python/lsst/dax/imgserv/api_soda.py

```diff
@@ -161,19 +175,33 @@ def imagesoda_unhandled_exceptions(error):
     return resp
 
 
+def _service_response(soda_method):
+    """ Return service info using DALI template.
+    """
+    service_ep = os.getenv("IMGSERV_FQDN", "localhost:5000")
```

Comment:
I thought we talked about using the Host header or you mentioned that flask had some special function to get the URL that you were called in on?

Reply:
It really is worth it to not have to configure this if at all possible, since it's always different everywhere, and the container will never know it.

Reply:
yep, changing to use url_for() in flask as for /capabilities and /examples, greatly simplifying the logic here.